pkg/utils/utils.go

```diff
@@ -0,0 +1,157 @@
+package utils
+
+import (
+  "math/rand"
+  "time"
+  "io/ioutil"
+  "github.com/sirupsen/logrus"
+
+  "k8s.io/api/core/v1"
+  apps "k8s.io/api/apps/v1"
+  metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+  route "github.com/openshift/api/route/v1"
+)
+
+func GetFileContents(filePath string) []byte {
+  contents, err := ioutil.ReadFile(filePath)
+  if err != nil {
+    logrus.Fatalf("Unable to read file to get contents: %v", err)
+  }
+
+  return contents
+}
+
+func init() {
+  rand.Seed(time.Now().UnixNano())
+}
+
+var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
+
+func GetRandomWord(wordSize int) []byte {
+  b := make([]rune, wordSize)
+  for i := range b {
+    b[i] = letters[rand.Intn(len(letters))]
+  }
+  return []byte(string(b))
+}
+
+func Secret(secretName string, namespace string, data map[string][]byte) *v1.Secret {
```

Comment:
Consider naming these functions like `newSecret` which I believe is the pattern to identify a 'constructor' like method.

Reply:
I would argue that this isn't a constructor though, rather we are returning an structure.